raiden/transfer/channel.py

```diff
@@ -1602,6 +1813,34 @@ def handle_refundtransfer(
     return is_valid, events, msg
 
 
+def handle_receive_lock_expired(
+        channel_state: NettingChannelState,
+        state_change: ReceiveLockExpired,
+):
+    """Remove expired locks from channel states."""
+    if state_change.transfer.channel_identifier != channel_state.identifier:
```

Comment:
this check is not necessary, it is being done by the `is_valid_lockexpired` function